manifests/rvm.pp

```diff
@@ -0,0 +1,122 @@
+# class hdm::rvm
+class hdm::rvm {
+  group { $hdm::group:
+    ensure => present,
+  }
+
+  user { $hdm::user:
+    ensure => present,
+    shell  => '/sbin/nologin',
+    home   => $hdm::hdm_path,
+    gid    => $hdm::group,
+  }
+
+  file { $hdm::hdm_path:
+    ensure => directory,
+    owner  => $hdm::user,
+    group  => $hdm::group,
+  }
+
+  include gnupg
+
+  include rvm
+
+  rvm::system_user { 'hdm':
+    create => false,
+  }
+
+  rvm_system_ruby { "ruby-${hdm::ruby_version}":
+    ensure      => present,
+    default_use => false,
+  }
+
+  rvm_gem { 'bundler':
+    ensure       => present,
+    ruby_version => "ruby-${hdm::ruby_version}",
+    require      => Rvm_system_ruby["ruby-${hdm::ruby_version}"],
+  }
+
+  # Fix for old g++ and sqlite3
+  case $facts['os']['family'] {
+    'RedHat': {
+      if versioncmp($facts['os']['release']['major'], '8') < 0 {
+        package { 'centos-release-scl':
+          ensure => present,
+        }
+
+        package { 'devtoolset-7':
+          ensure => present,
+        }
+        $exec_prefix = 'scl enable devtoolset-7 '
+        exec { 'update sqlite':
+          command => 'yum install -y https://kojipkgs.fedoraproject.org//packages/sqlite/3.8.11/1.fc21/x86_64/sqlite-devel-3.8.11-1.fc21.x86_64.rpm https://kojipkgs.fedoraproject.org//packages/sqlite/3.8.11/1.fc21/x86_64/sqlite-3.8.11-1.fc21.x86_64.rpm',
```

Comment:
I think we can use the package resource and provide the link to the package?

Reply:
No. we need to install both packages at once as we are upgrading from sqlite 3.7 to 3.8.